.travis.yml

```diff
@@ -0,0 +1,162 @@
+# READ THIS
+# The Travis CI environment needs to have the following defined:
+# 1. CLOWDER_HOST_URI - the Clowder host URI to test against. Needs to be clean for each testing run (no post-
```

Comment:
is this needed? isn't this just http://localhost:9000/ ?

Reply:
It's not strictly necessary at this point. I was thinking that in the future it could be changed to point to a different instance. I'd have it as a variable anyway since it's used in many places

Reply:
is it possible in travis to use default values, so if people don't set them in their fork it will still work?

Reply:
See also: https://drive.google.com/open?id=1XluZEZNzYHW6rDXrJOrddhE0lG9lTj1-JvRoKjiN6Kg